### ISSUE TITLE
A .ddl file is detected as virus or unwanted software by windows defender

### ISSUE THREAD

**Issue by TheSecondSpoon** [bcssov/IronyModManager#255]
**Describe the bug**
The following file is detected by windows defender as virus or unwanted software:

Avalonia.Win32.dll

**Version:**
 - OS: Windows 10 
~~- Irony Version 1.16.26~~

**Comment by Gamma-Andromedae** [comment]
I just had the same issue.
Windows defender detected 2 files as Trojan:Win32/Emali.A!cl:

Avalonia.Win32.dll
Avalonia.Themes.Default.dll

**Version:**

- OS: Windows 10 Pro, version 10.0.19042 Build 19042
- Irony Version 1.16.26

**Comment by bcssov** [comment]
It's a false positive and an upstream issue as well. Add an exclusion and when the guys who made avalonia fix this I'll be sure to include it in the updated version.

**Comment by bcssov** [comment]
So per Avalonia team 0.10.3 version is affected as a false positive, 0.10.4 is ok. And I've personally tested this. This means that you guys are **not** on Irony 1.16.26 but on 1.16.17 or older.

If you don't want to whitelist Irony then upgrade to 1.16.26.

**Comment by Gamma-Andromedae** [comment]
Sorry to reply to a closed issue, just wanted to clarify. 
I'm fairly confident I was using the latest version, as I only started using Irony about 10 hours before I commented on this post and it worked perfectly the multiple times I launched it then. It was only after I'd restarted my PC and tried to launch than that windows flagged the files as trojans. 
I downloaded the zip (portable) version and replaced the 2 dll files that got flagged from that. No issues since then.
